src/win/rlimit.c

```diff
@@ -129,99 +118,79 @@ int setrlimit(int resource, const struct rlimit *rlp)
   if(rlp->rlim_cur < rlimits[resource].rlim_max)
     rlimits[resource].rlim_cur = rlp->rlim_cur;
   else
-    iRet = EINVAL;
+    return EINVAL;
   //
   // We should not let the user set the max value.  However,
   // since currently there is no defined source for initial
   // values, we will let the user change the max value.
   //
   rlimits[resource].rlim_max = rlp->rlim_max;
 
-  return iRet;
+  return 0;  // success
 }
 
 /////////////////////////////////////////////////////////////////
 // Wrap the real fwrite() with this rfwrite() function, which is
 // resource limit aware.
 //
-//
 ///////////////////////////////////////////////////////////////size_t
 size_t rfwrite(const void *buffer, size_t size, size_t count, FILE *stream)
 {
-  long position;
-  size_t written;
-  __int64 liByteCount, liPosition;
   //
   // Convert the count to a large integer (64 bit integer)
   //
-  liByteCount = (__int64)count;
+  __int64 liByteCount = (__int64)count;
 
   //
   // Get the current file position
   //
-  position = ftell(stream);
-  liPosition = (__int64)position;
+  __int64 liPosition = (__int64)ftell(stream);
 
   //
   // Check to make sure the write will not exceed the RLIMIT_FSIZE limit.
   //
-  if((liPosition + liByteCount) > rlimits[RLIMIT_FSIZE].rlim_cur)
-  {
+  if(liPosition + liByteCount > rlimits[RLIMIT_FSIZE].rlim_cur)
     //
```

Comment:
If I read correctly the diff I would keep the comments + return 0 inside { } for clarity.

Reply:
Actually, I would incline to remove most of of the comments completely. Comments are useful to provide broader context information that is not easily to infer from the code itself. Code comments should not explain the code's obvious functionality as it is the case here. Code should be self-explanatory as far as possible.

Nevertheless, I will just add the braces again.